CodeSniffer/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -382,7 +406,7 @@ protected function processParams(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $co
                         $suggestedTypeHint = 'callable';
                     } else if (in_array($typeName, PHP_CodeSniffer::$allowedTypes) === false) {
                         $suggestedTypeHint = $suggestedName;
-                    } else if (PHP_VERSION_ID >= 70000) {
+                    } else if ($this->_platform >= 70000) {
```

Comment:
I've updated the pull request with the only other place I can add the check, PSR1 ClassDeclarationSniff